client/packages/core/src/data/task-data.ts

```diff
@@ -0,0 +1,56 @@
+export enum Responsible {
+  ROLE = 'Role',
+  ROLE_ATTR = 'Role form Attr.',
+  USER_ATTR = 'User form Attr.',
+  NOBODY = 'Nobody & delete'
+}
+
+export enum CustomFieldType {
+  STRING = 'String',
+  TEXT = 'Text',
+  NUMBER = 'Number',
+  TIMESTAMP = 'Timestamp'
+}
+
+export interface CustomField {
+  name: string;
+  type: CustomFieldType;
+  value: string;
+}
+
+export interface TaskData {
+  config: {
+    persist: boolean;
+    task: {
+      name: string;
+      description: string;
+      category: string;
+      priority: string;
+      expiry: {
+        timeout: string;
+        error: string;
+        responsible: {
```

Comment:
I think we should align here: as @ivy-lmu is soon implementing this in the backed: 
I thought that another structure may be easier:

```
responsible: {
  mode = "role/user/..." <-- enum
  assignee = "Everybody/System/... " or "in.myUser" ...
}
```
What do you think about it?

Reply:
Yes this is the current state from the backend.
And yes something you proposed should be the future :+1:

Reply:
the first one. It's ok if you use the second one to save, but it's really difficult to check which value is available and what to render for it...

Reply:
good, yes I think the first one would be the best. Naturally users will think of the data being structured as it is presented on the UI ... keeping the 'mode' and the 'assignee' separate, is also on the plain JSON easier to use. As it is less easy to setup a corrupt configuration :)